scipy/special/_basic.py

```diff
@@ -3420,6 +3425,11 @@ def zeta(x, q=None, out=None):
     see [dlmf]_ for details. The Riemann zeta function corresponds to
     the case when ``q = 1``.
 
+    For complex inputs with ``q = None`, points with
+    ``abs(z.imag) > 1e9` and 49 < abs(z.real) < 50 are currently not
```

Comment:
It's not that Euler-Maclaurin can't do it, just that it takes so long that the function will hang indefinitely for sufficiently large imaginary part. I figure it would be better to return NaN then have something take hours to years to compute. This should probably be documented better though.

Reply:
Yeah, from what I recall, EM had trouble for large imaginary components near the critical strip, but that doesn't seem to be documented here. The trouble for real component near 50 was different IIRC.